code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -209,6 +210,24 @@
 		holder.remove_reagent(/datum/reagent/consumable/capsaicin, 2)
 	..()
 
+//See block comment in ../milk/overdose_process(mob/living/M) for calculation and explanation of why this exists and why 5 was chosen
+/datum/reagent/consumable/milk/overdose_start(mob/living/M)
+	M.reagents.add_reagent(/datum/reagent/toxin/bonehurtingjuice, 5)
+	. = ..()
```

Comment:
```suggestion
	return ..()
```
When you're calling this at the end of the proc, you can just use `return ..()`, which is a little bit cleaner. Same goes for basically any return value